stytch/user.go

```diff
@@ -43,3 +43,11 @@ func (c *Client) DeleteUser(userID string) (*DeleteUserResponse, error) {
 	err := c.newRequest("DELETE", path, nil, &retVal)
 	return retVal, err
 }
+
+func (c *Client) DeleteUserEmail(userID string, email string) (*DeleteUserEmailResponse, error) {
+	path := "/users/" + userID + "/emails/" + email
+
+	var retVal *DeleteUserEmailResponse
+	err := c.newRequest("DELETE", path, nil, &retVal)
```

Comment:
the other ones are being fixed in another open pr but would you mind updating this one
```suggestion
	err := c.newRequest("DELETE", path, nil, retVal)
```